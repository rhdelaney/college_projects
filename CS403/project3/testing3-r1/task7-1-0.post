(println "trying (twinPrimes 6)")
(stream-display 27 (twinPrimes 6))
(println)
