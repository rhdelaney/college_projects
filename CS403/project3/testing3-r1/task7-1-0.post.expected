trying (twinPrimes 6)
((2 3) (2 5) (2 7) (3 5) (3 7) (5 7) (5 11) (7 11) (7 13) (11 13) (11 17) (13 17) (13 19) (17 19) (17 23) (19 23) (23 29) (29 31) (31 37) (37 41) (37 43) (41 43) (41 47) (43 47) (47 53) (53 59) (59 61) ...)
